content/01.abstract.md

```diff
@@ -2,6 +2,13 @@
 
 Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
 However, traditional research and multi-author writing processes break down at scale.
-New tools and workflows that rely on automation can catch errors and ensure that contributions are fairly tracked in massively collaborative research.
-We present techniques for overcoming challenges of open research, with special emphasis on manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and many other aspects of manuscript building.
+We present a new software tool named Manubot, available at <https://manubot.org>, for open scholarly writing.
+Manubot adopts the contribution workflow used by many large-scale open source software projects to enable collaborative authoring of scholarly manuscripts.
+With Manubot, manuscripts are written in Markdown and stored in a Git repository to precisely track changes over time.
+By hosting manuscript repositories publicly, such as on GitHub, multiple authors can simultaneously propose and review changes.
+Publication with Manubot is continuous.
+When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
+Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers, whose metadata is then retrieved and rendered to a specified style.
```

Comment:
`persistent identifiers` is also not a familiar term.  Can we add an example or two?

We also have `rendered` in consecutive sentences.

Reply:
`rendered` → `displayed` should be fine. And consider `specified` → `user specified` so that it's clear that it is customizable.